src/ngraph/node.cpp

```diff
@@ -298,6 +298,16 @@ void Node::set_friendly_name(const string& name)
     m_friendly_name = name;
 }
 
+size_t Node::get_opset_version() const
```

Comment:
Opset is a property of the node implementation, not its instantiation. This should be a virtual function.